releases/nightly-build/compose-files/source/Makefile

```diff
@@ -28,6 +28,8 @@ $(eval $(ARGS):;@:)
 ifeq (dev, $(filter dev,$(ARGS)))
 	CORE_EDGEX_REPOSITORY=edgexfoundry
 	DEV:=-dev
+	# matches the way that edgex-go to versioning the docker images for `make docker`
+	CORE_EDGEX_VERSION:=$(shell cat ./VERSION 2>/dev/null || echo 0.0.0)
```

Comment:
There will never be a VERSION files in this folder. Just use `CORE_EDGEX_VERSION=0.0.0` as we know that is what is used for local builds.

Reply:
ok